doc/developer-documentation/getting-started.md

```diff
@@ -29,7 +29,7 @@ In a project folder:
 aeproject node
 ```
 
-To spwan a temporary local chain, use `aeproject node --stop` to stop it.
+To spawn a temporary local chain, use `aeproject node --stop` to stop it.
```

Comment:
This sentence doesn't make any sense. Should it something like this "To stop already spawned local node use ..." ?